### ISSUE TITLE
[Bug] Playnext list fails to load

### ISSUE THREAD

**Issue by SparrowBrain** [SparrowBrain/Playnite.PlayNext#22]
```
26-01 19:51:11.478|INFO |PlayNext#GameActivityExtension:6 games with recent activity found
26-01 19:51:11.746|INFO |PlayNext#HowLongToBeatExtension:1145 games with HowLongToBeat data found
26-01 19:51:11.758|DEBUG|PlayNext#GameActivityExtension:Games with playtime: 6
26-01 19:51:11.801|INFO |PlayNext#HowLongToBeatExtension:1145 games with HowLongToBeat data found
26-01 19:51:11.801|DEBUG|PlayNext#GameActivityExtension:Games with playtime: 6
26-01 19:51:12.097|ERROR|PlayNext#PlayNext:Failure while parsing activities
System.InvalidOperationException: Sequence contains no matching element
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source, Func`2 predicate)
   at PlayNext.Model.Score.TotalScoreCalculator.<>c__DisplayClass14_0.<Calculate>b__0(KeyValuePair`2 score) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\TotalScoreCalculator.cs:line 78
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at PlayNext.Model.Score.TotalScoreCalculator.Calculate(PlayNextSettings savedSettings) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\TotalScoreCalculator.cs:line 76
   at PlayNext.PlayNext.<<RefreshPlayNextData>b__39_0>d.MoveNext() in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\PlayNext.cs:line 212
26-01 19:51:12.138|ERROR|PlayNext#PlayNext:Failure while parsing activities
System.InvalidOperationException: Sequence contains no matching element
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source, Func`2 predicate)
   at PlayNext.Model.Score.TotalScoreCalculator.<>c__DisplayClass14_0.<Calculate>b__0(KeyValuePair`2 score) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\TotalScoreCalculator.cs:line 78
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at PlayNext.Model.Score.TotalScoreCalculator.Calculate(PlayNextSettings savedSettings) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\TotalScoreCalculator.cs:line 76
   at PlayNext.PlayNext.<<RefreshPlayNextData>b__39_0>d.MoveNext() in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\PlayNext.cs:line 212
```

**Comment by SparrowBrain** [comment]
Couple of issues:
* failure to read HLTB or GameActivity date should not prevent showing the results;
* It looks like the calculation is invoked too many times at the startup.

**Comment by SparrowBrain** [comment]
Partially fixed in v1.5.0:
* 26-01 19:51:12.097|ERROR|PlayNext#PlayNext:Failure while parsing activities - the message was wrong. It's not happening while parsing activities
* Removed unnecessary calculation on startup.

Should still investigate, why sequence contained no elements, and probably prevent a crash there.

**Comment by SparrowBrain** [comment]
This is most likely fixed.
